pkg/connector/host_group.go

```diff
@@ -162,11 +162,18 @@ func (r *hostGroupResourceType) Entitlements(ctx context.Context, resource *v2.R
 	}
 
 	if bag.Current() != nil && bag.Current().ResourceTypeID == hbacRuleEntryType { // Dynamic entitlements for host group hbac rules
+		hostGroupDN := resource.GetExternalId().GetId()
+		if hostGroupDN == "" {
+			return nil, "", nil, fmt.Errorf("baton-ipa: host group resource %s has no external ID", resource.DisplayName)
```

Comment:
Do we just want to return no entitlements in this case, and not return an error?